<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>941</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="x_val_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>113</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>значение х</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_val_input">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>113</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>значение у</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_point_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Искать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Введенные точки</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="points_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>321</width>
      <height>831</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="del_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="del_num_input">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>121</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер точки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="task_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Условие</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>731</width>
      <height>761</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
